frontend/src/metabase/components/Schedule/Schedule.module.css

```diff
@@ -2,25 +2,33 @@
   line-height: 2.5rem;
   display: grid;
   grid-template-columns: fit-content(100%) auto;
-  gap: 0.5rem;
-  row-gap: 0.35rem;
+  gap: var(--schedule-grid-gap, 0.5rem);
+  row-gap: var(--schedule-row-gap, 0.35rem);
+  font-weight: var(--schedule-font-weight, normal);
 }
 
 .ControlGroup {
   display: flex;
   gap: 0.25rem;
-  flex-flow: row wrap;
+  flex-flow: row nowrap;
+  flex-shrink: 0;
   grid-column: 2;
+  align-items: center;
 }
 
 .TextInFirstColumn {
   display: flex;
   flex-flow: row wrap;
-  justify-content: flex-end;
   align-items: flex-start;
+  font-weight: var(--schedule-first-column-font-weight, normal);
 }
 
 .TextInSecondColumn {
   line-height: 1.5rem;
   grid-column: 2;
+  font-weight: var(--schedule-second-column-font-weight, normal);
```

Comment:
I did:

```
rg schedule-seco
```

but didn't find any hits except this line

Reply:
This is indeed not used anywhere at the moment, but we will integrate this component to Subscriptions sidebar later, and we'll have to make all fonts bolder in there. So I've decided to introduce both variables right away.

Reply:
Not a blocker as far as I'm concerned, but my sense is that there's a general preference not to have undefined CSS variables, since that's usually undesired/buggy behavior, even though, I suppose it doesn't cause problems when a default value is provided. We even have a script that checks for undefined CSS variables, though I believe the script is not used.